api/watson_service.py

```diff
@@ -21,7 +21,10 @@ def analyze_tone(mood_text):
               content_type='application/json',
           ).get_result()
   
-  raw_watson_response = result['document_tone']['tones'][0]
-  watson_result = raw_watson_response['tone_name']
+  if result['document_tone']['tones']:
+    raw_watson_response = result['document_tone']['tones'][0]
+    watson_result = raw_watson_response['tone_name']
+  else:
+    watson_result = "Unable to analyze"
```

Comment:
This conditional checks if the tone array returned a result before attempting to access it, otherwise just returns "unable to analyze"